### PR TITLE
Resize CPU with custom cpu share

### PR DESCRIPTION
**Pull request by santhosh-sloka**
What this PR does / why we need it:

The code will help successful CPU resize Operation by preserving the custom CPU share deployed by the customer.

Which issue(s) this PR fixes(unless minor typo change ,better to have an issue opened):

https://github.ibm.com/zvc/planning/issues/18693

Special notes for your reviewer:

**There is a dependent PR for this in icic-dev repo, " https://github.ibm.com/zvc/icic-dev/pull/6088 "  both of the PR's should be merged together**

Results are attached here :

https://github.ibm.com/zvc/planning/issues/18693#issuecomment-88412796
https://github.ibm.com/zvc/planning/issues/18693#issuecomment-88412883 
https://github.ibm.com/zvc/planning/issues/18693#issuecomment-88499911 << resize with VM shutdown

[optional] which components (compute, zvm/kvm etc)

zvm

[optional] backward incompatible change

[optional] security impact

[optional] open source package impact

--signed-off : santhoshs.1988@gmail.com ( ssanthosh@in.ibm.com )

### CODE REVIEW

**Comment by santhosh-sloka** [comment]
@jichenjc , @bjhuangr , @SeanHQF - Please help review the PR, thanks..

File: zvmsdk/api.py [hunk 1]
```diff
@@ -1005,7 +1005,7 @@ def guest_create(self, userid, vcpus, memory, disk_list=None,
                                          rdomain, pcif)
 
     @check_guest_exist()
-    def guest_live_resize_cpus(self, userid, cpu_cnt):
+    def guest_live_resize_cpus(self, userid, cpu_cnt, cpu_share=''):
```
Comment (bjhuangr) [inline zvmsdk/api.py:1008]: update feilong doc accoordingly for the new parameter

Comment (santhosh-sloka) [reply zvmsdk/api.py:1008]: sure @bjhuangr , will update the feilong doc as well

**Review by SeanHQF — APPROVED** [review]

**Review by bjhuangr — APPROVED** [review]

**Comment by bjhuangr** [comment]
@santhosh-sloka please update the doc in another PR

**Comment by santhosh-sloka** [comment]
> @santhosh-sloka please update the doc in another PR

sure @bjhuangr ,Thanks  i am working on it, will update the doc and raise the new PR soon

**Pull request merged**